media/themes/Modern/modern.msstyles/modern.rc

```diff
@@ -444,3 +444,6 @@ DARK_USERTILEBACKGROUND_BMP BITMAP  "bitmaps/Dark/DARK_USERTILEBACKGROUND.bmp"
 #ifdef LANGUAGE_ZH_CN
     #include "lang/zh-CN.rc"
 #endif
+#ifdef LANGUAGE_UA_UA
```

Comment:
There is no language such as "UA_UA", did you mean "UK_UA". (This applies to every translation made)